definitions/infra-harbor/prometheus_dashboard.json

```diff
@@ -0,0 +1,221 @@
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": true
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT sum(harbor_project_repo_total) FROM Metric FACET project_name SINCE 24 hours ago LIMIT 10 "
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Project quota consumption (%)",
+            "layout": {
+              "column": 5,
+              "row": 5,
+              "width": 4,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "dataFormatters": [
+                {
+                  "name": "Utilization",
+                  "precision": 1,
+                  "type": "decimal"
+                }
+              ],
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT (latest(harbor_project_quota_usage_byte)/1073741824)/(latest(harbor_project_quota_byte)/1073741824) * 100 as Utilization FROM Metric FACET project_name ORDER BY Utilization LIMIT 10 SINCE 1 hours ago UNTIL now"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "thresholds": [
+                {
+                  "alertSeverity": "WARNING",
+                  "value": 50
+                },
+                {
+                  "alertSeverity": "CRITICAL",
+                  "value": 85
+                }
+              ]
+            }
+          },
+          {
+            "title": "In-flight requests",
+            "layout": {
+              "column": 9,
+              "row": 5,
+              "width": 4,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.area"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT latest(harbor_core_http_inflight_requests) FROM Metric SINCE 60 MINUTES AGO TIMESERIES"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          }
+        ]
+      }
+    ],
+    "variables": []
```

Comment:
```
[
  {
    "instancePath": "",
    "schemaPath": "#/additionalProperties",
    "keyword": "additionalProperties",
    "params": {
      "additionalProperty": "variables"
    },
    "message": "must NOT have additional properties"
  }
]
```

Reply:
er .. explain?

Reply:
should I knock off the variables block altogether?

Reply:
yes, could you please remove it?